docs/v1/P4Runtime-Spec.mdk

```diff
@@ -4496,7 +4496,7 @@ multi-reader lock (also known as multiple-readers/single-writer lock).
 Conversely (&eg; in a single-threaded architecture), it may choose to serialize
 `Read` RPC processing.
 
-## Parallelism of read and write requests
+## Parallelism of Read and Write requests
```

Comment:
Actually that's not specific to "read" and "write", we capitalize every significant word in titles: https://github.com/p4lang/p4runtime/tree/master/docs#what-to-capitalize

Reply:
Doh!  If I had read your comment more carefully, I would have noticed that.  Fixed now.